packages/ui-docs-client/src/Header/index.js

```diff
@@ -39,14 +44,96 @@ import theme from './theme'
 class Header extends Component {
   static propTypes = {
     name: PropTypes.string.isRequired,
-    version: PropTypes.string
+    version: PropTypes.string,
+    versionsData: PropTypes.shape({
+      latestVersion: PropTypes.string.isRequired,
+      previousVersions: PropTypes.arrayOf(PropTypes.string).isRequired
+    })
   }
 
   static defaultProps = {
-    version: undefined
+    version: undefined,
+    versionsData: undefined
   }
 
+  handleSelect = (e, [selectedVersion]) => {
+    const { versionsData } = this.props
+    const { latestVersion } = versionsData
+    const isSelectedLatestVersion = selectedVersion === latestVersion
+    const splittedUrl = window.location.pathname.split('/').filter(Boolean)
```

Comment:
Webstorm shows a typo here, should probably be "splitUrl", or "pathNameParts"

Reply:
Renamed it to `pathNameParts`